csrc/executor.cpp

```diff
@@ -262,6 +263,16 @@ void FusionExecutor::compileFusion(
     return;
   }
 
+  // NOTE: Profiling needs to be started below the isExpressionEvaluated query
+  // given the conditional can exit early from compilation.
```

Comment:
What happens when a fusion contains multiple segments with one of them going through expression evaluator?

Reply:
The primary concern of capturing compile time in the `FusionExecutor` is capturing the nvrtc compile time for a code generated kernel given ExpressionEvaluator is not compiling a kernel..  Expression Evaluation would be  captured for a fusion at the `FusionExecutorCache` level.